packages/dev/gui/src/2D/controls/image.ts

```diff
@@ -291,6 +292,24 @@ export class Image extends Control {
         }
     }
 
+    /**
+     * Gets or sets the referrer policy to apply on the img load request,
+     * you should set this field before set the source field if you want to ensure the header will be present on the xhr loading request
+     */
+    @serialize()

```

Comment:
Not sure if this should be serialized ???

@RaananW ???

Reply:
That's an interesting one. 
I'll comment about the PR in general, but in this specific instance, if it made sense to merge this PR then it is fine to serialize it.